File: cfgs/pythoncfgs/FIND_FIRST_NATURAL_NUMBER_WHOSE_FACTORIAL_DIVISIBLE_X.py
Nodes
1;statement;foo_1 = 1;8;
2;statement;foo_1 = 1;9;
3;conditional;foo < foo;10;
4;statement;return foo_1;14;
5;statement;foo_1 = foo_2 * foo_3;11;
6;conditional;foo_1 % foo_2 == 0;12;
7;exit;;
Edges
0;2;
1;3;
2;4;5;
3;7;
4;6;
5;3;4;
6;

File: cfgs/java_cfgs_foo/FIND_FIRST_NATURAL_NUMBER_WHOSE_FACTORIAL_DIVISIBLE_X.java
Nodes
1;special;;;
2;statement;int i = 1;14;
3;statement;int fact = 1;15;
4;statement;i = 1;16;
5;conditional;foo_1 < foo_2;17;
6;statement;fact = foo_1 * foo_2;19;
7;conditional;(foo_1 % foo_2) == 0;20;
8;statement;break;20;
9;statement;i++;18;
10;statement;return foo_1;22;
11;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;
4;6;10;
5;7;
6;8;9;
7;10;
8;5;
9;11;
10;
